src/views/Dashboard/Lab/index.vue

```diff
@@ -39,16 +39,16 @@
               inline
               color="primary"
               :size="20"
-              v-if="labAccount.verificationStatus == 'Verified'"
+              v-if="labAccount && labAccount.verificationStatus == 'Verified'"
             >mdi-check-decagram</v-icon>
             <v-icon 
               inline
               color="yellow darken-2"
               :size="20"
               v-else
             >mdi-information</v-icon>
-            <b v-if="labAccount.verificationStatus == 'Verified'">Your account has been verified</b>
-            <b v-else>Your account has not been verified</b>
+            <b v-if="labAccount && labAccount.verificationStatus == 'Unverified'">Your verification submission is being reviewed by Daogenic</b>
```

Comment:
@bukandicki can you re-check the wording from Sarah / Haidar? If it's already correct, i just give a little input that the name should be DAOGenics, not Daogenic

Reply:
Hi @hilyds, according to Haidar, the wording is correct. Of course, I can change Daogenic to DAOGenics as soon as possible.
![image](https://user-images.githubusercontent.com/67890542/152086953-6481545b-b9f5-4db4-a7e2-3858206b38bd.png)